lib/apachetest.pm

```diff
@@ -42,7 +42,9 @@ sub setup_apache2 {
     my %args = @_;
     my $mode = uc $args{mode} || "";
     # package hostname is available on sle15+ and openSUSE, on <15 it's net-tools
-    my @packages = qw(apache2 /bin/hostname);
+    my @packages = qw(apache2 hostname);
```

Comment:
Using `hostname` breaks compatibility with SLE < 15. See the comment and acb78c757364495d1e9a33464199e60008f9003f.